Makefile

```diff
@@ -5,3 +5,8 @@ build:
 unbuild:
 	rm -rf lib
 	mv src lib
+
+publish:
```

Comment:
I'd add some sort of a check to see that it actually built correctly before publishing, also unbuild should really make sure that the src directory exists before it removes lib